ui/packages/catalog-ui-search/src/main/webapp/js/model/MetacardAction.js

```diff
@@ -12,6 +12,8 @@
 var Backbone = require('backbone');
 require('backbone-associations');
 const URITemplate = require('urijs/src/URITemplate');
+const ENCODED_QUERY_ID_TEMPLATE = '%7B%26queryId%7D';
```

Comment:
✏️ Can be replaced with encodeURIComponent(DECODED_QUERY_ID_TEMPLATE)

Reply:
Yep, I like that better. Thanks.